internal/postprocessor/main.go

```diff
@@ -150,7 +149,12 @@ func (c *config) run(ctx context.Context) error {
 		log.Println("exiting post processing early")
 		return nil
 	}
-	if err := gocmd.ModTidyAll(c.googleCloudDir); err != nil {
+	// TODO(codyoss): In the future we may want to make it possible to be able
+	// to run this locally with a user defined remote branch.
+	if err := c.AmendPRDescription(ctx); err != nil {
```

Comment:
I think it may be best to keep this at the end of the logic if possible. That way we don't update the PR if other steps failed. Thoughts?

Reply:
I see now why you movded it up. I would say we should refactor out the bit we need to do earlier(module dection),

Reply:
Separated it so that the PR Title and Description are stored in the config struct and set at the beginning of running the postprocessor. Writing to file occurs separately as a last step in the postprocessing.